src/main/java/de/muenchen/isi/api/dto/AbfrageDto.java

```diff
@@ -35,6 +36,9 @@ public class AbfrageDto {
 
     private StatusAbfrage statusAbfrage;
 
+    @ReadOnlyProperty
```

Comment:
Was wird mit der Annotation `org.springframework.data.annotation.ReadOnlyProperty` bezweckt?

Es handelt sich um eine Annotation welche in einer Entität verhindert, dass dieses Attribut persistiert wird.

https://docs.spring.io/spring-data/commons/docs/current/api/org/springframework/data/annotation/ReadOnlyProperty.html

Reply:
Da es sich bei der AbfrageDto um ein DTO für lesende Zugriffe handelt, kann die Annotation wegfallen.

Falls bei einem DTO bestimmte Attribute nur serialisiert aber nicht deserialisert werden sollen, dann ist die FasterXml-Property `@JsonProperty(access = JsonProperty.Access.READ_ONLY)` zu verwenden.

Reply:
siehe auch: https://github.com/it-at-m/isi-backend/pull/102#discussion_r1345364683